### DISCUSSION TITLE
Error message: Yang & Smith-RBGV pipeline

### DISCUSSION THREAD

**Discussion by a-phang** [chrisjackson-pellicle/Yang-and-Smith-paralogy-resolution#4] [Q&A] [answered]
Hello,
Have run the Hybpiper-RBGV pipeline with no issues (thank you!), but when moving on to the Yang and Smith one to address paralogs, the job keeps failing because I keep hitting the error below. Even though I have Singularity, Nextflow and python loaded on the server (Linux), and followed the instructions in the tutorial same as for Hybpiper.

```
Error executing process > 'CHECK_AND_BATCH_PARALOGS_01 (1)'
Caused by:
  Process `CHECK_AND_BATCH_PARALOGS_01 (1)` terminated with an error exit status (2)
Command executed:
  python /Yang-and-Smith-RBGV-scripts/01a_check_outgroups_and_batch.py     11_paralogs
Command error:
  python: can't open file '/Yang-and-Smith-RBGV-scripts/01a_check_outgroups_and_batch.py': [Errno 2] No such file or directory
```
I've tried both pointing to the latest sif and using the original config, but I still get the same error. Am I missing something?

Grateful for any advice, thank you!

**Comment by chrisjackson-pellicle** [comment, marked as answer]
Hi @a-phang,

Many apologies for the late reply - I only just saw this now!

This was my fault - when I last made changes (quite a few months ago now) I forgot to update the tag on the Singularity container, so pulling the `:latest` was actually pulling an old version...

Should be fixed now, but you'll need to download the Singularity container again. Let me know how you go!

Cheers,

Chris

**Reply by a-phang** [reply]
It all works now with the new Singularity container, thank you Chris!
